common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java

```diff
@@ -0,0 +1,332 @@
+    @Test
+    public void testTimeMillisToHumanString() {
+        PowerMockito.mockStatic(System.class);
+        PowerMockito.when(System.currentTimeMillis())
+                .thenReturn(1558787014000l);
+
+        Assert.assertEquals("20190525132334000",
+                UtilAll.timeMillisToHumanString());
+    }
+
+    @Test
+    public void testTimeMillisToHumanStringLong() {
+        Assert.assertEquals("20190525132334000",
+                UtilAll.timeMillisToHumanString(1558787014000l));
+    }
+
+    @PrepareForTest({System.class, UtilAll.class})
+    @Test
+    public void testComputNextMorningTimeMillis() {
+        PowerMockito.mockStatic(System.class);
+        PowerMockito.when(System.currentTimeMillis())
+                .thenReturn(1515585600000L);
+
+        Assert.assertEquals(1515628800000l,
+                UtilAll.computNextMorningTimeMillis());
+    }
+
+    @PrepareForTest({System.class, UtilAll.class})
+    @Test
+    public void testComputNextMinutesTimeMillis() {
+        PowerMockito.mockStatic(System.class);
+        PowerMockito.when(System.currentTimeMillis())
+                .thenReturn(1515585600000L);
+
+        Assert.assertEquals(1515585660000l,
+                UtilAll.computNextMinutesTimeMillis());
+    }
+
+    @PrepareForTest({System.class, UtilAll.class})
+    @Test
+    public void testComputNextHourTimeMillis() {
+        PowerMockito.mockStatic(System.class);
+        PowerMockito.when(System.currentTimeMillis())
+                .thenReturn(1515585600000L);
+
+        Assert.assertEquals(1515589200000l,
+                UtilAll.computNextHourTimeMillis());
+    }
+
+    @PrepareForTest({UtilAll.class, System.class})
+    @Test
+    public void testComputNextHalfHourTimeMillis() {
+        PowerMockito.mockStatic(System.class);
+        PowerMockito.when(System.currentTimeMillis())
+                .thenReturn(1515585600000L);
+
+        Assert.assertEquals(1515591000000l,
+                UtilAll.computNextHalfHourTimeMillis());
+    }
+
+    @Test
+    public void testTimeMillisToHumanString2() {
+        Assert.assertEquals("2019-04-25 13:23:34,000",
+                UtilAll.timeMillisToHumanString2(1556195014000l));
+    }
+
+    @Test
+    public void testTimeMillisToHumanString3() {
+        Assert.assertEquals("20190425132334",
+                UtilAll.timeMillisToHumanString3(1556195014000l));
+    }
+
+    @Test
+    public void testGetDiskPartitionSpaceUsedPercent() {
+        Assert.assertEquals(-1,
+                UtilAll.getDiskPartitionSpaceUsedPercent(""), 0);
+        Assert.assertEquals(-1,
+                UtilAll.getDiskPartitionSpaceUsedPercent(null), 0);
+        Assert.assertEquals(-1,
+                UtilAll.getDiskPartitionSpaceUsedPercent("foo"), 0);
+    }
+
+    @Rule
+    private TemporaryFolder folder = new TemporaryFolder();
+
+    @Test
+    public void testGetDiskPartitionSpaceUsedPercent1() throws IOException {
+        File file = folder.newFile("foo.txt");
```

Comment:
Do you consider the data clearance when test stateful function?

Reply:
@vongosling I have added a finally block that deletes the file and TemporaryFolder. Is that ok? I am not entirely sure about that